### PR TITLE
Dynamic Call

### LINKED ISSUE #2173: Neo doesn't compile against Neo VM master branch
```
C:\Users\harry\Source\neo\official\3neo-monorepo\core\src\neo\SmartContract\ApplicationEngine.cs(197,40): 
error CS7036: There is no argument given that corresponds to the required formal parameter 'rvcount' of 'ExecutionEngine.CreateContext(Script, ushort, int, int)' 
[C:\Users\harry\Source\neo\official\3neo-monorepo\core\src\neo\neo.csproj]
C:\Users\harry\Source\neo\official\3neo-monorepo\core\src\neo\SmartContract\ApplicationEngine.cs(360,39): 
error CS1503: Argument 2: cannot convert from 'int' to 'Neo.SmartContract.CallFlags' [C:\Users\harry\Source\neo\official\3neo-monorepo\core\src\neo\neo.csproj]   
```

### PR DESCRIPTION
**Pull request by erikzhang**
Close #2173

### CODE REVIEW

**Comment by erikzhang** [comment]
Can someone fix the UTs? 😅

**Comment by erikzhang** [comment]
Wait for #2168

**Comment by shargon** [comment]
I will fix the UT

**Comment by erikzhang** [comment]
@shargon Now we can move on.

**Comment by Tommo-L** [comment]
UT failed

**Comment by shargon** [comment]
Give me one day to fix the UT

File: src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs [hunk 1]
```diff
@@ -58,6 +58,7 @@ partial class ApplicationEngine
             [OpCode.CALL] = 1 << 9,
             [OpCode.CALL_L] = 1 << 9,
             [OpCode.CALLA] = 1 << 9,
+            [OpCode.CALLT] = 1 << 9,
```
Comment (erikzhang) [inline src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:61]: It's too cheap. Maybe `1 << 15`.

Comment (shargon) [reply src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs:61]: done

**Comment by shargon** [comment]
@superboyiii could you test it?

**Comment by superboyiii** [comment]
> @superboyiii could you test it?

Sure

**Comment by superboyiii** [comment]
I've tested vote, register, make Txs via send command and invoke nep5 methods like balanceOf, decimals and totalSupply. They all work well.

**Comment by erikzhang** [comment]
Go?

**Review by shargon — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Pull request merged**